typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -281,12 +277,14 @@ delete $p has $e;
 insert $p has email "kevin2@gmail.com";
 ----
 
+Updates the new email "kevin@gmail.com" for every person with a full name that contains "Kevin"
+to be "kevin2@gmail.com" instead.
+It does that by deleting ownership over attribute with old value
```

Comment:
There should be alarms ringing in your head when you see _two_ pronouns with unclear antecedents (namely, "it" and "that"). Please rephrase.